docs/guide/docs/nlu/improvement.md

```diff
@@ -0,0 +1,138 @@
+---
+id: improvement
+title: Improving Your NLU
+---
+
+## Bot Improvement Module
+Botpress provides a way for you to check if your QnA module's AI is performing well. You can do this by using the Bot Improvement module. This module lets the chatbot user rate responses from the chatbot using a simple thumbs-up/thumbs-down scale. These appear as two icons at the bottom left of the message bubble containing an answer from the QnA Module.
+
+![Thumb Icons](../assets/b-i-chat.png)
+
+### Module Interface
+The bot improvement interface contains all the QnA items, which the user rated with a thumbs-down. These appear in the `Feedback Items` section. For context, a preview of the conversation is displayed when you select a feedback item.
+
+![Graphical User Interface](../assets/b-i-interface.png)
```

Comment:
I would use something else than shut up here. A softer insult such as ''You useless chatbot'' or ''dumb bot''